manifests/params.pp

```diff
@@ -253,6 +253,8 @@
 
   # Application logging
   $logging_level = 'info'
+  $logging_type = 'file'
+  $logging_layout = 'pattern'
```

Comment:
Default was `multiline_pattern`, but I am fine with switching to `pattern`. The difference is that the former adds ` |` characters at the beginning of each line, which is useful for development but in syslog/journald this might not be the best thing to do. So +1.

Reply:
Was `multiline_pattern` also used before structured logging was introduced? I thought it was `pattern` so stayed with it here.